Plugins/XMLHttpRequest/CMakeLists.txt

```diff
@@ -1,8 +1,21 @@
-set(SOURCES 
-    "Include/Babylon/XMLHttpRequest.h"
-    "Source/XMLHttpRequest.cpp"
-    "Source/XMLHttpRequest.h"
-    "Source/XMLHttpRequest${BABYLON_NATIVE_PLATFORM}.cpp")
+if(APPLE)
```

Comment:
If we're splitting the implementation down here, too, could the networking code just live here?

Reply:
You mean remove NetworkUtils from BabylonNativeUtils and put it in XMLHttpRequest project?

Reply:
Kind of. I was more thinking about just having the network code itself be here as a non-exposed thing, similar to how UWP does it. I'd forgotten that the script loader needs network stuff too, though, so it probably wouldn't work now that I think of it.